include/sdf/parser.hh

```diff
@@ -144,13 +238,27 @@ namespace sdf
   /// string it is converted to SDF first. All strings are converted to the
   /// latest SDF version
   /// \param[in] _xmlString XML string to be parsed.
-  /// \param[in] _sdf Pointer to an SDF object.
+  /// \param[out] _sdf Pointer to an sdf Element object.
```

Comment:
nit Case "sdf" is inconsistent with other usages "SDF"

(Here and elsewhere)

Reply:
e42bbda